core/domain/feedback_jobs_continuous.py

```diff
@@ -44,15 +44,20 @@ class FeedbackAnalyticsAggregator(jobs.BaseContinuousComputationManager):
 
     @classmethod
     def get_event_types_listened_to(cls):
+        """Returns a list of event types that this class subscribe to."""
```

Comment:
When a function returns something, you should describe it in the docstring.
For more info, check out our [coding style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#python) with regards to writing docstring and more!